src/containers/StructurePage/folderComponents/menuOptions/PublishTopic.tsx

```diff
@@ -69,56 +85,57 @@ const PublishTopic = ({ t, id, contentUri }: Props) => {
 
   const publishTopic = () => {
     if (!done) {
+      fetchTopicArticle(id, locale)
```

Comment:
Fikk ikkje med meg at denne ikkje var med i publiseringa i første omgang...

Reply:
Den var det, men da brukte jeg bare contentUri. Jeg skrev om publishResource til at den tar inn en Resource og ikke en string. Jeg må derfor gjøre et ekstra kall for emne. Når jeg tenker meg om så går det sikker an å sette sammen en Resource fra props, men det er kanskje ryddigere som det er.

Reply:
Synes det såg ryddig ut.